bin/burden

```diff
@@ -209,6 +211,10 @@ gl_cpu_type_request="none"
 gl_run_file=""
 gl_failed_test_rpt="failed_tests"
 
+#
+# Make sure the lock directory exists
+#
+mkdir -p ${gl_top_dir}/lock_dir
```

Comment:
Does this `lock_dir` ever get cleaned up?  I don't see a `rmdir` in this PR

Reply:
No, as we have no clue when we are the last one, also you may start another burden when that one is finishing up.